### DISCUSSION TITLE
Svelte 🦾

### DISCUSSION THREAD

**Discussion by zhendalf** [tailwindlabs/headlessui#486] [Ideas]
Please add Svelte components, alongside with Vue and React 🙏

**Comment by AndrewEQ** [comment]
Svelte is the way of the future! 😄

**Reply by suneetk92** [reply]
Why Svelte? Why not LitElement or stenciljs?

**Reply by jonbitgood** [reply]
Different Goals, stenciljs still has runtime / uses a virtual DOM while svelte has no virtual DOM and no runtime.

**Comment by grandslammer** [comment]
+1

**Comment by rgossiaux** [comment]
For anyone following this thread who may be interested, I've just released an unofficial full Svelte port of this library: https://github.com/rgossiaux/svelte-headlessui

Same components with the same API and same implementation. Hope this helps!

**Reply by xWayfinder** [reply]
Would love to see this get folded into the offical library!

**Reply by zhendalf** [reply]
Just seeing it now, but beautiful work @rgossiaux!

**Reply by oravecz** [reply]
Wonder why there are crickets from the maintainers in most of these discussions?

**Comment by joshistoast** [comment]
+1
